src/day_3.rs

```diff
@@ -0,0 +1,39 @@
+use std::fs::File;
+use std::io::{self};
+
+const N_BITS: usize = 12;
```

Comment:
This fails the sample data which is 5 bits

-$10,000 from your salary

Reply:
Damn it my salary is negative now